osi_common.proto

```diff
@@ -154,6 +154,9 @@ message BaseStationary
 
     /// The relative orientation of the landmark w.r.t its parent frame.
     optional Orientation3d orientation = 3;
+    
+    /// The two dimensional (flat) contour of the object. 
```

Comment:
@RudolfVocilka 
Hi, I just discussed this issue with @CarloVanDriestenBMW on the phone. We use the contour in a purely two dimensional context. That's why I also defined a vector2d in order to avoid confusion about the z coordinate. In a 2D world there is no elevation.

I agree with solving the reference frame discussion elsewhere.